src/main/java/org/commcare/formplayer/services/RestoreFactory.java

```diff
@@ -218,6 +220,7 @@ public UserSqlSandbox performTimedSync(boolean shouldPurge, boolean skipFixtures
                         extras
                 );
             } catch (InvalidCaseGraphException e) {
+                FormplayerSentry.captureException(e, SentryLevel.INFO);
```

Comment:
I think this might be a warning level event since we think it should never happen.